virttest/libvirt_xml/domcapability_xml.py

```diff
@@ -0,0 +1,120 @@
+"""
+Module simplifying manipulation of XML described at
+http://libvirt.org/formatdomaincaps.html
+"""
+
+from virttest import xml_utils
+from virttest.libvirt_xml import base, accessors, xcepts
+
+
+class DomCapabilityXML(base.LibvirtXMLBase):
+
+    """
+    Handler of libvirt domcapabilities operations.
+
+    """
+    __slots__ = ('features')
+    __schema_name__ = 'domcapabilities'
+
+    def __init__(self, virsh_instance=base.virsh):
+        accessors.XMLElementNest(property_name='features',
+                                 libvirtxml=self,
+                                 parent_xpath='/',
+                                 tag_name='features',
+                                 subclass=DomCapFeaturesXML,
+                                 subclass_dargs={
+                                     'virsh_instance': virsh_instance})
+
+        super(DomCapabilityXML, self).__init__(virsh_instance)
+        self['xml'] = self.__dict_get__('virsh').domcapabilities().stdout.strip()
+
+
+class DomCapFeaturesXML(base.LibvirtXMLBase):
+    """
+    Handler of feature element in libvirt domcapabilities.
+
```

Comment:
And here, it's better to add the 'Properties' here, you can check capability_xml.py